app/services/teacher_training_public_api/sync_courses.rb

```diff
@@ -117,5 +114,23 @@ def create_new_accredited_provider(accredited_body_code, recruitment_cycle_year)
 
       accredited_provider
     end
+
+    def notify_of_new_course!(provider, accredited_provider_code)
+      notification = [":seedling: #{provider.name}, which has courses open on Apply, added a new course"]
+      accredited_provider = ::Provider.find_by(code: accredited_provider_code)
+
+      if accredited_provider&.onboarded?
+        notification << "It’s ratified by #{accredited_provider.name}, who have signed the DSA"
+      elsif accredited_provider.present?
+        notification << "It’s ratified by #{accredited_provider.name}, who have NOT signed the DSA"
+      else
+        notification << 'There’s no separate accredited body for this course'
+      end
+
+      SlackNotificationWorker.perform_async(
+        notification.join('. ') + '.',
+        support_interface_provider_courses_path(provider),
```

Comment:
This needs to be `support_interface_provider_courses_url`

Reply:
Good catch! I must have messed up the rebase. Will fix